stdlib/private/StdlibCollectionUnittest/CheckSequenceType.swift

```diff
@@ -427,6 +446,11 @@ public let filterTests = [
   ),
 ]
 
+public let containsTests = [
```

Comment:
You can use `findTests` dataset for testing `contains()` (we do it for `Sequence.contains()`).  Just check for `FindTest.expected != nil`.